src/poc/miner_poc_statem.erl

```diff
@@ -75,10 +75,12 @@
     retry = ?CHALLENGE_RETRY :: non_neg_integer(),
     receipts_timeout = ?RECEIPTS_TIMEOUT :: non_neg_integer(),
     poc_restarts = ?POC_RESTARTS :: non_neg_integer(),
-    txn_ref = make_ref() :: reference()
+    txn_ref = make_ref() :: reference(),
+    addr_hash_filter :: undefined | {pos_integer(), pos_integer(), pos_integer(), binary(), bloom_nif:bloom()}
```

Comment:
can we do a module local record for this instead of a tuple?

Reply:
Yep